packages/common/src/fields/File.ts

```diff
@@ -0,0 +1,52 @@
+import Field from './Field'
+
+type AllowedDisks = "public" | "private"
```

Comment:
We do not have disks in our application. So let the `File` class have two methods instead: `public()` to upload publicly, and `private()` to upload privately.